src/events/dns.js

```diff
@@ -0,0 +1,239 @@
+const uuid4 = require('uuid4');
+const dns = require('dns');
+const shimmer = require('shimmer');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const serverlessEvent = require('../proto/event_pb.js');
+const eventInterface = require('../event.js');
+const errorCode = require('../proto/error_code_pb.js');
+
+// BLACKLIST
+const URL_BLACKLIST = {
+    'tc.epsagon.com': 'endsWith',
+};
+/**
+ * Checks if a URL is in the blacklist
+ * @param {string} url The URL to check
+ * @param {string} path The Path to check (optional)
+ * @returns {boolean} True if it is in the blacklist, False otherwise
+ */
+function isBlacklistURL(url, path) {
```

Comment:
maybe move to a common helper? (not sure it's the same function signature but if it is i'd recommend it)

Reply:
Nice, I added helpers/events.js and converted http&dns events to use it.